docs/installation-and-operation/information-collection.md

```diff
@@ -1,104 +1,29 @@
 ---
-title: About the information we collect
+title: About the anonymous usage data we collect
 redirect_from:
   - /docs/latest/information-collection
 ---
 
-# About the information we collect
+# About the anonymous usage data we collect
 
-Metabase uses Google Analytics and Snowplow to collect anonymous usage information from the installed servers that enable this feature. Below is a representative list of the events we have instrumented, as well as the information we collect about the user performing the action and the instance being used.
+If you've opted in to provide us with anonymous usage data (thank you!), Metabase will phone home some data collected via Google Analytics and Snowplow.
```

Comment:
Pretty sure we don't use GA anymore ([asked here](https://metaboat.slack.com/archives/C05MPF0TM3L/p1710768894438609))

Reply:
We do, :/